client/src/components/EntityTypes.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Form, FormControl, FormLabel } from 'react-bootstrap';
+import _ from 'lodash';
+
+const EntityTypes = (props) => {
+	const errorsArr = _.get(props.errors, props.name);
+    const options=[{value: 'select'}, {value: 'other'}];
+
+
+	return (
+		<div className="form-group">
+				<FormLabel className={props.className} style={inputStyle}>{props.title}</FormLabel>
+
+				<FormControl
+					isInvalid={props.errors && errorsArr}
+					as={props.as}
+					multiple
+					id={props.name}
+					name={props.name}
+					value={props.value}
+					onChange={props.handleChange}
+					>
+                    <option value="" disabled>
+                            Select
+                    </option>
+                    {
+                        props.options.map((option) => {
+                            return (<option key={option.id} value={option.id}>{option.name}</option>)
+                        })
+                    }
+				</FormControl>
+
+			{errorsArr && (
+				<FormControl.Feedback type="invalid">
+					{errorsArr.map((error, index) => (
+						<div key={`field-error-${props.name}-${index}`} className="fieldError">
+							{error}
+						</div>
+					))}
+				</FormControl.Feedback>
+			)}
+		</div>
+	);
+};
+const inputStyle = {
+	color: '#124E54'
```

Comment:
Can we move this into a CSS file and reference a class from there?

Reply:
Done!